### PR TITLE
Ethereal Jaunt is faster

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Maybe this time it will be picked more compared to blink, which covers a larger overall distance than jaunt does.
This makes jaunt have the normal movement speed that most players have. It's more fun this way, given the cooldown
Neat escape tool too.
It's unironically fun
By the way this is just wizard jaunt, doesn't apply to other jaunt spells like vampire jaunt or wraith jaunt.
:cl:
 * tweak: The ethereal jaunt wizard spell had its speed increased, and is as fast as regular movement.

### CODE REVIEW

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
>Maybe this time it will be picked more

#22949

Jaunt is already picked by nearly 40% of ALL wizards and is more popular than Blink

**Comment by boy2mantwicethefam** [comment]
It is a sub-par spell compared to blink

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
Every spell is sub-par compared to blink

**Comment by boy2mantwicethefam** [comment]
Pain mirror is great

File: code/modules/spells/targeted/ethereal_jaunt.dm [hunk 1]
```diff
@@ -28,12 +29,12 @@
...
 		mass_jaunt(targets, duration, enteranim, exitanim, mist)
 	else
-		ethereal_jaunt(targets[1], duration, enteranim, exitanim, mist)
+		ethereal_jaunt(targets[1], duration, enteranim, exitanim, mist, movement_tally)
 
-/proc/ethereal_jaunt(var/mob/living/target, duration, enteranim = "liquify", exitanim = "reappear", mist = 1)
+/proc/ethereal_jaunt(var/mob/living/target, duration, enteranim = "liquify", exitanim = "reappear", mist = 1, movement_tally)
 	var/mobloc = get_turf(target)
 	var/previncorp = target.incorporeal_move //This shouldn't ever matter under usual circumstances
-	if(target.incorporeal_move == INCORPOREAL_ETHEREAL) //they're already jaunting, we have another fix for this but this is sane
```
Comment (jknpj) [inline code/modules/spells/targeted/ethereal_jaunt.dm:36]: Check if this define is still being used anywhere, if not, remove it.

Comment (jknpj) [reply code/modules/spells/targeted/ethereal_jaunt.dm:36]: Or you can just avoid magic numbers and make a new define for 1 and leave this as define in the child spells.

Comment (boy2mantwicethefam) [reply code/modules/spells/targeted/ethereal_jaunt.dm:36]: Used by mass jaunt, which I didn't touch
I swapped incorporeal_move with a variable named movement_tally that can be var-edited for further spells. 
While searching for instances of incorporeal_ethereal I have found an interaction for it for areas that do not allow ethereal jaunting, and I will fix this when I get around to do it

**Comment by Diavill** [comment]
ah yes, the diety treatment

**Comment by Caboozles** [comment]
The problem is nothing stops you from using jaunt AND blink.
And don’t say they have total overlap. Jaunting into the medical storage room is a wizard pastime.

**Comment by SonixApache** [comment]
how about
 - making this an upgrade to jaunt
 - adding a lesser jaunt that's innate to wizards so people don't NEED to pick jaunt or blink

**Comment by Kurfursten** [comment]
I'd rather not homogenize wizard movement. I think not being sure about what kind of displacement spell wizards have is interesting gameplay. I'd think it'd be nice if there was a third alternative to blink/jaunt though.

**Comment by boy2mantwicethefam** [comment]
I was thinking it should be an empowerment tbh

**Comment by SonixApache** [comment]
the innate ability to plink through a single wall or something would be good imo

**Comment by Kurfursten** [comment]
The cult wall shortcut runes are a bit like that.

I was thinking it would be appropriately wizardy if we had a Knightly Leap. Activate, it highlights all tiles one L shape away from you (like a chess knight) as the blueprints do. Click on one to move there instantly. Shortish cooldown. It's different from blink and jaunt because you have to click your target location, more predictable than blink for enemies but also more controlled. Not as controlled or unpredictable as jaunt.

**Comment by boy2mantwicethefam** [comment]
Also do not merge this yet, I have to work on making it not ignore anti-ethereal zones

**Comment by SonixApache** [comment]
it's been 19 days since your last comment, did you manage to fix it yet

**Comment by boy2mantwicethefam** [comment]
I'm gonna look into it sooner or later, not too much of a priority because I need to gather myself to actually manage to code this, only PJB and maybe some other coders know how gliders work

**Comment by boy2mantwicethefam** [comment]
I'll work on this at a later point, but the idea won't die

**Pull request closed without merging**